src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/Processors/LedgerTransactionMarkers/TransactionTypeLedgerTransactionMarkerProcessor.cs

```diff
@@ -81,13 +81,17 @@ public TransactionTypeLedgerTransactionMarkerProcessor(ProcessorContext context)
 
     public void VisitTransaction(CoreModel.CommittedTransaction committedTransaction, long stateVersion)
     {
+        if (committedTransaction.LedgerTransaction is CoreModel.RoundUpdateLedgerTransaction)
```

Comment:
VERY MINOR: If it's possible in C Sharp, maybe it would be slightly more clear that we've covered all cases if we inline the return into the match as:
```csharp
CoreModel.RoundUpdateLedgerTransaction => {
    // We purposefully don't tag round updates, because we don't allow filtering on them.
    // And as there are quite a few round updates, this improves ingestion and querying performance.
    return;
}
```

(Also with a comment explaining why)

Reply:
nope, it's not possible to exit from function when you're inside switch expression, it has to return some value.